server/service/integration_mdm_test.go

```diff
@@ -359,7 +359,7 @@ func (d *device) scepEnroll() {
 	require.NoError(t, err)
 
 	notBefore := time.Now()
-	notAfter := notBefore.Add(time.Hour * 1)
```

Comment:
😅 I initially tried with `time.Hour * 12` and this was a leftover, thanks

Reply:
ha, I didn't remember making that change :D That was probably just a reflex, there was nothing wrong with that, just my preference showing!